src/pages/document/[id].tsx

```diff
@@ -283,6 +301,17 @@ const FamilyPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({ pa
               )}
             </section>
           ))}
+
+          {!!geographySummary && (
+            <section className="mt-8" data-cy="top-documents">
+              <DocumentSearchForm
+                placeholder={`Search the full text of ${totalDocsInPageGeography} documents from ${geographyName}`}
```

Comment:
I realise this would be a logical impossibility apart from on the geo page if the given geo has no documents in it - so safe to leave it as-is for now!

Reply:
Have changed anyway - good to explicitly hide!